ktor-client/ktor-client-core/common/src/io/ktor/client/plugins/sse/SSE.kt

```diff
@@ -49,6 +49,10 @@ public val SSE: ClientPlugin<SSEConfig> = createClientPlugin(
         LOGGER.trace("Sending SSE request ${request.url}")
         request.setCapability(SSECapability, Unit)
 
+        if (getAttributeValue(request, sseRequestAttr) != true) {
```

Comment:
Can we add this check before `LOGGER.trace("Sending SSE request ${request.url}")`?